packages/editor/package.json

```diff
@@ -41,9 +41,7 @@
     "test:watch": "jest --env=jsdom --verbose --watch",
     "test-coverage": "yarn test -- --coverage",
     "show-coverage": "yarn test-coverage && coverage\\lcov-report\\index.html",
-    "start-with-no-browser": "set BROWSER=none&&react-scripts start",
-    "start-cypress": "yarn run cypress open",
-    "test:cypress": "npm-run-all --parallel start-with-no-browser start-cypress"
+    "start-with-no-browser": "set BROWSER=none&&react-scripts start"
```

Comment:
what the new script do? do we need it?

Reply:
I just removed the comma here from line 44.  No new script.